core/src/global/_utility-classes.scss

```diff
@@ -389,7 +389,8 @@
 }
 
 .tds-u-flex-center {
-  justify-content: space-between !important;
+  display: flex !important;
+  justify-content: center !important;
   align-items: center !important;
```

Comment:
To me justify center and align center makes more sense for this type of utility class.